src/lib/media-query/match-media-provider.ts

```diff
@@ -0,0 +1,39 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+import {
+  NgZone,
+  PLATFORM_ID,
+  InjectionToken,    // tslint:disable-line:no-unused-variable
```

Comment:
You can avoid suppressing the lint error by explicitly using `InjectionToken` in the place where it is inferred